tools/user_migration/migrate_user.py

```diff
@@ -193,6 +199,19 @@ def get_fxa_id(databases, user):
     return (fxa_kid, fxa_uid)
 
 
+def create_migration_table(database):
+    try:
+        cursor = database.cursor()
+        cursor.execute(
+            """create table if not exists migration (
+                fxa_uid varchar(255) NOT NULL PRIMARY KEY,
```

Comment:
With this being for the python syncstorage side to reference, I think it will be much easier to use its "legacy" `userid` vs syncstorage-rs's `fxa_uid`

Reply:
Yeah, but "userid" is kinda db dependent since [it can come off of the db row](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/assignment/sqlnode/sql.py#L368-L396). fxa_uid isn't. I'd rather there be zero chance of collision here, so I'd prefer to use fxa_uid.

Reply:
Cool, I forgot it's really not difficult to get at either (it's already setup on the user dict/object "fxa_uid" field).